experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -290,9 +294,9 @@ describe('fetch', () => {
   });
 
   describe('when request is successful', () => {
-    beforeEach(done => {
+    beforeEach(async () => {
```

Comment:
Is it still required to make all of these async?

Reply:
The tests would still work, I think, but it's nicer to await on the completion of the execution of `api.context.with`, then to call `done` before it finishes from inside.